pkg/platform/abstract/platform.go

```diff
@@ -402,6 +404,48 @@ func (ap *Platform) CreateProject(createProjectOptions *platform.CreateProjectOp
 	return platform.ErrUnsupportedMethod
 }
 
+// EnrichCreateProjectConfig enrich project configuration with defaults
+func (ap *Platform) EnrichCreateProjectConfig(createProjectOptions *platform.CreateProjectOptions) error {
+
+	// transform display name if needed
+	if !createProjectOptions.SkipTransformDisplayName {
+		if createProjectOptions.ProjectConfig.Spec.DisplayName != "" {
+
+			// name is UUID
+			if _, err := uuid.ParseUUID(createProjectOptions.ProjectConfig.Meta.Name); err == nil {
+				ap.transformDisplayNameToName(createProjectOptions.ProjectConfig)
+				ap.Logger.DebugWith("Project name has been transformed",
+					"name", createProjectOptions.ProjectConfig.Meta.Name)
```

Comment:
log previous

Reply:
a warn-level log line exists in `transformDisplayNameToName`